packages/office-ui-fabric-react/src/components/DetailsList/DetailsRow.tsx

```diff
@@ -197,6 +197,7 @@ export class DetailsRow extends React.Component<IDetailsRowProps, IDetailsRowSta
         data-selection-index={ itemIndex }
         data-item-index={ itemIndex }
         data-is-draggable={ isDraggable }
+        draggable= { isDraggable }
```

Comment:
@dzearing any idea who might know more about `data-is-draggable` in case it's really not used (haven't checked)?